### PR TITLE
refactor(Cron): update expression initialization

### PR DESCRIPTION
**Pull request by kogel-net**
# Description

Fix cron expression initialization

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: src/Masa.Blazor/Presets/Cron/Items/CronItem.razor.cs [hunk 1]
```diff
@@ -122,14 +108,6 @@ protected override Task OnAfterRenderAsync(bool firstRender)
...
                 _dayOfWeeks = Enum.GetValues<DayOfWeek>().ToList();
             }
-
-            _periodStart = Period == PeriodTypes.Year ? DateTime.Now.Year : _minUnit;
-            _periodEnd = Period == PeriodTypes.Year ? DateTime.Now.AddYears(10).Year : 2;
-            _startFromPeriod = Period == PeriodTypes.Year ? DateTime.Now.Year : _minUnit;
-            _startEveryPeriod = 1;
-            _nearestOfDay = _minUnit;
-            _lastPeriodOfWeek = _minUnit;
-            _selectWeekNumber = WeekNumbers.First;
```
Comment (capdiem) [inline src/Masa.Blazor/Presets/Cron/Items/CronItem.razor.cs:132]: 这些初始化丢了没关系吗？

File: src/Masa.Blazor/Presets/Cron/Items/CronItem.razor [hunk 2]
```diff
@@ -22,29 +22,29 @@
...
+                    @I18n.T($"$masaBlazor.cron.range.from") <MTextField Value="_cronItemData.PeriodStart" HideDetails="true" Outlined Dense ValueChanged="OnPeriodStartChanged" Type="number" NumberProps="prop=> { prop.HideControl = true; prop.Min = _minUnit; prop.Max = _cronItemData.PeriodEnd; }" TValue="int?"></MTextField> @I18n.T("$masaBlazor.cron.range.to") <MTextField Value="_cronItemData.PeriodEnd" TValue="int?" Type="number" NumberProps="prop=> { prop.HideControl = true; prop.Min= _cronItemData.PeriodStart; prop.Max = _maxUnit; }" Dense Outlined HideDetails="true" ValueChanged="OnPeriodEndChanged"></MTextField> @_i18nUnitText
```
Comment (capdiem) [inline src/Masa.Blazor/Presets/Cron/Items/CronItem.razor:25]: CronItem.razor 文件格式化一下。

**Review by capdiem — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Review by capdiem — APPROVED** [review]
LGTM

**Pull request merged**